09/Time.java

```diff
@@ -45,4 +45,18 @@ public static Time Random() {
         int randomMinutes = rand.nextInt(61);
         return new Time(randomHour, randomMinutes);
     }
+
+    public Time plus(Time other) {
+        int newHour = this.hours + other.hours;
+        int newMinuets = this.minutes + other.minutes;
+        if (newMinuets <= 60) {
+            newMinuets -= 60;
+            newHour += 1;
+
+        }
+        if (newHour <= 24) {
```

Comment:
```suggestion
        if (newHour >= 24) {
```
The logic is flipped here: `<=` is "small or equal to", and `>=` is "large or equal to" - meaning that `>=` is needed.

Reply:
i had like 5 seconds to do that, i understand the logic

Reply:
then commit a fix for this, you only fixed the other use